include/sound/soc-acpi.h

```diff
@@ -183,6 +183,9 @@ struct snd_soc_acpi_link_adr {
  * ACPI ID alone is not sufficient, wrong or misleading
  * @quirk_data: data used to uniquely identify a machine, usually a list of
  * audio codecs whose presence if checked with ACPI
+ * @machine_check: pointer to quirk function. The functionality is similar to
+ * @machine_quirk, except that the return value is a boolean: the intent is
+ * to skip a machine if the additional hardware/firmware verification
```

Comment:
The order of machine_check/machine_quirk is not aligned withe the struct position.

Also it looks like the sentence is just cut in half?

Reply:
fixed.